pages/glossary/changes-in-content.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Changes in content
+key: changes-in-content
+unambiguous: true
+objective: true
+input_aspects:
+  - Accessibility tree
+  - Audio output
+  - CSS styling
+  - DOM tree
+---
+
+A _change in the content_ of a [web page][] caused by an event occurs when, by comparing the [web page][] before and after the event firing, at least one of the following occurs:
```

Comment:
There is something in the causality that creates confusion. As in, there are two different concepts mixed here:
1. the page changes in one of the 3 ways indicated.
2. the page changes as a reaction to an event.

This does creates confusion in the assumptions. Notably:
* time-based media => it causes changes but not changes due to an event, hence the need to specify "if there's time based media, ignore it".
* changes only caused by the event => which is a bit weird as you pointed out, but becomes clearer and/or redundant if de-coupled from the even causing it. Currently, any "change in the content" has to be caused by an event (or at least perceived as such) which make the assumption that there are no changes not caused by event weird… because this use the other meaning of change…

Maybe it is worth splitting the definition in two ("changes" and "changes caused by an event") or at least to somehow streamline the difference between the two concepts.

Reply:
But this definition is about "change in the content of a web page caused by an event"

I don't think it helps to have a definition of just change (because change implies comparing something, which implies having to define what is going to be compared).

However, I do think it will help if we focus on dom events, given that this definition if being used in rules that deal with dom events. Hopefully, this will make it easier to address the issues.